submitted_models/marble_husky_sensor_config_3/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Please remove editor support files `.vscode/*`

Reply:
A commit removing this on my fork has been made.